### DISCUSSION TITLE
Apple Privacy Manifest

### DISCUSSION THREAD

**Discussion by galyataneva** [capacitor-community/bluetooth-le#654] [Q&A] [answered]
Hello, Apple mandates that app developers now specify approved reasons for API usage to enhance user privacy. Can you give information what is the required dictionary key and the recommended reason for using the Bluetooth-le. This information will be  included in PrivacyInfo.xcprivacy?

**Comment by pwespi** [comment, marked as answer]
Currently Apple lists the following privacy relevant APIs that require a reason:
- NSPrivacyAccessedAPICategoryFileTimestamp
- NSPrivacyAccessedAPICategorySystemBootTime
- NSPrivacyAccessedAPICategoryDiskSpace
- NSPrivacyAccessedAPICategoryActiveKeyboards
- NSPrivacyAccessedAPICategoryUserDefaults
https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api

This plugin does not use any of those APIs.

**Reply by pwespi** [reply]
However there is still `NSBluetoothAlwaysUsageDescription` that needs to be set in `Info.plist`, see https://github.com/capacitor-community/bluetooth-le#ios.

**Reply by galyataneva** [reply]
Thank you.
